backend/substrapp/tests/views/tests_views_algo.py

```diff
@@ -207,6 +207,10 @@ def test_algo_create(self):
                 'permissions': {
                     'public': True,
                     'authorized_ids': [],
+                },
+                'metadata': {
+                    "one": "foo",
+                    "two": "bar"
```

Comment:
It looks like this is never tested in the return value. Since just adding these lines doesn't do anything maybe we should leave this file untouched?

Reply:
I agree, I'll remove this!